src/date-picker/date-picker-inline.jsx

```diff
@@ -43,19 +51,15 @@ const DatePickerInline = React.createClass({
       ...other,
     } = this.props;
 
-    if (!open) {
-      return <span />;
-    }
-
     return (
       <div style={styles.container}>
-        <div style={styles.subContainer}>
-          <Paper {...other}>
+        <div style={styles.subContainer} ref="anchor">
+          <Popover open={open} anchorEl={this.refs.anchor} {...other}>
```

Comment:
Can you put the {...other} at the beginning? I think that it's better in general, you don't want specific properties to be overridden.